tests/tinderbox_scraper/test_tinderbox_scraper.py

```diff
@@ -142,6 +142,20 @@
      'target': 'mozilla-central-firefox-25.0a1.en-US.linux-i686.txt',
      'target_url': 'firefox/tinderbox-builds/mozilla-central-linux/'
                    '1374583608/firefox-25.0a1.en-US.linux-i686.txt'},
+    # -a firefox -t tinderbox -p win32 --extension=txt
+    {'args': {'application': 'firefox',
+              'extension': 'txt',
+              'platform': 'win32'},
+     'target': 'mozilla-central-firefox-39.0a1.en-US.win32.txt',
```

Comment:
I don't understand this. I see many different versions in the tests. I will change it, but I don't see the reasoning. Doing a "find . -type f" from the tests/data folder, I see versions 21, 23, 25, 37.0b1, 27.0a1, 37, and 38.0a1.